internal/install/install.go

```diff
@@ -184,9 +187,14 @@ func writeKubernetesDeployerResources(elasticPackagePath *locations.LocationMana
 		return errors.Wrap(err, "can't read application configuration")
 	}
 
+	elasticAgentManagedYaml, err := DownloadFile("elastic-agent-managed-kubernetes.yaml", elasticAgentManagedYamlUrl)
```

Comment:
WDYT to replace this function with less generic one like `downloadElasticAgentManagedYAML()`? I don't think we need generic `DownloadFile` function here.